### ISSUE TITLE
Adding an accessibility menu when tabbing to allow user to select what item should be focused

### ISSUE THREAD

**Issue by ahmadayubi** [Maps4HTML/Web-Map-Custom-Element#264]
Would adding a menu that is only accessible when tabbing be good for accessibility, similar to how Google search has a menu of links, first one being able to control what element is in focus next.

In the case of mapml viewer it would be a menu of what should be focused next i.e. features, zoom controls, layer controls, etc.

https://user-images.githubusercontent.com/55214462/106053646-09872480-60b9-11eb-937b-ffd97799c500.mp4

@Malvoz is a menu like this standard and warranted?

**Comment by prushforth** [comment]
@malvoz, @shepazu Is this accessibility menu part of a Web standard for accessibility?

**Comment by Malvoz** [comment]
That's a convention sometimes referred to as "skip links": https://webaim.org/techniques/skipnav/.
Here's an example: https://genericcomponents.netlify.app/generic-skiplink/demo/index.html.

It works by using a CSS hack to visually hide the links and unhide them as they recieve focus. I call it a hack because there is no standard way to hide something that's intended to be focusable (related discussions, FWIW: https://github.com/w3c/csswg-drafts/issues/560, https://github.com/w3c/css-a11y/issues/13).

It's indeed helpful, it relates to <span title="Success Criterion">SC</span> [2.4.1 Bypass Blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks). I suppose I can see a standard implementation of it (both as a CSS property and as a default facility in maps) but I think that's also based on assumptions of how users currently interact with maps (and what is keyboard accessible, under which circumstances).

**Comment by Malvoz** [comment]
I accidentally stumbled upon a demo found in https://github.com/Canadian-Geospatial-Platform/GeoView which uses a menu like that. I don't believe a native implementation should need such a menu but thought I'd report the find nevertheless.

**Comment by Malvoz** [comment]
I believe navigation can be made intuitive and accessible through proper DOM order and keyboard shortcuts, along with the navigation controls in popups. Hopefully even more so through https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/396 and https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/397.

A menu such as described here should not be necessary. I'm closing this, feel free to continue discussing or re-open.
